docs/content/development/extensions-core/simple-client-sslcontext.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: doc_page
+---
+
+## Simple SSLContext Provider Module
+
+This module contains a simple implementation of [SSLContext](http://docs.oracle.com/javase/8/docs/api/javax/net/ssl/SSLContext.html)
+that will be injected to be used with HttpClient that Druid nodes use internally to communicate with each other. To learn more about
+Java's SSL support, please refer to [this](http://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/JSSERefGuide.html) guide.
+
+# Configuration
+
+|Property|Description|Default|Required|
+|--------|-----------|-------|--------|
+|`druid.client.https.algorithm`|SSL Algorithm/Protocol to use.|none|yes|
```

Comment:
Hm, the SSLContext documentation for getInstance calls the parameter a "protocol" but throws `NoSuchAlgorithmException` and points to the `SSLContext Algorithms` section you linked

In that case, I think either protocol/algorithm is fine, but it would be good if each property in this section had a direct link to the StandardNames section that corresponds with the property.